src/utilities/frame/hooks.tsx

```diff
@@ -0,0 +1,27 @@
+import {useContext} from 'react';
+import {useAppBridge} from '../app-bridge';
+import {FrameContext} from './context';
+
+export function useFrame() {
+  const frame = useContext(FrameContext);
+  const appBridge = useAppBridge();
+
+  if (!frame && !appBridge) {
```

Comment:
Should this be or rather than and?

If either the Frame or the AppBridge is defined then you can use this